src/components/Shared/ReportIncidentPage/index.js

```diff
@@ -0,0 +1,247 @@
+import React, { Component } from 'react';
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+
+import { logError } from 'services/logger';
+
+import ContentPage from 'components/Shared/ContentPage';
+import ButtonPrimary from 'components/Shared/Button/ButtonPrimary';
+import ButtonSecondary from 'components/Shared/Button/ButtonSecondary';
+import TextArea from 'components/Shared/Form/TextArea';
+import Checkbox from 'components/Shared/Form/Checkbox';
+
+import IncidentLocationModal from './IncidentLocationModal';
+import './ReportIncidentPage.css';
+
+const Required = () => (
+  <span className="ReportIncidentPage__required">Required*</span>
+);
+const Optional = () => (
+  <span className="ReportIncidentPage__optional">Optional</span>
+);
+
+function toggleArrayItem(array, item) {
+  if (array.includes(item)) {
+    return array.filter(el => el !== item);
+  }
+  return [...array, item];
+}
+
+class ReportIncidentPage extends Component {
+  constructor(...args) {
+    super(...args);
+
+    this.state = {
+      notes: '',
+      description: '',
+      latitude: null,
+      longitude: null,
+      communityIds: [],
+      errorMessage: '',
+      loading: false,
+      usersNotifiedCount: null,
+      createdReport: null,
+      isLocationModalOpen: false,
+    };
+  }
+
+  onSubmit = async e => {
+    try {
```

Comment:
It'd be cool to refactor the backend to not need to have the /incident/report endpoint, and instead create a user with report incident permissions only and use graphql directly from this component. Definitely beyond the scope of this PR but worth making an issue for.